codalab/model/bundle_model.py

```diff
@@ -1616,6 +1623,11 @@ def add_worksheet_items(self, worksheet_uuid, items, after_sort_key=None, replac
             if len(items) == 0:
                 # Nothing to insert, return
                 return
+
+            # if after_sort_key not specified, insert the bundle at the end of the worksheet
```

Comment:
Provide more comments / context for this: going forward, this code should not be triggered, etc.